django/core/files/utils.py

```diff
@@ -24,6 +27,8 @@ def __init__(self, file):
     write = property(lambda self: self.file.write)
     writelines = property(lambda self: self.file.writelines)
     xreadlines = property(lambda self: self.file.xreadlines)
+    if six.PY3:
```

Comment:
do we need six.PY3? I assume AttributeError will be raised on Python 2 either way.